RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -389,8 +389,10 @@ protected void executeRun(ExecutionContext context, boolean preventLoggingOnQuer
                         context.execute();
                         finallyStartJobsOfContext(context);
                     }
-                } finally {
+                } catch(Exception ex) {
```

Comment:
space before (  

(I'm only commenting on this because this is a pretty frequent change in the recent commits)